src/network.c

```diff
@@ -1683,6 +1685,42 @@ static int network_set_interface(const sockent_t *se,
   return 0;
 } /* }}} network_set_interface */
 
+static int network_bind_socket_to_addr(sockent_t *se,
+                                       const struct addrinfo *ai) {
+
+  if (se->data.client.bind_addr == NULL)
+    return 0;
+
+  DEBUG("network_plugin: fd %i: bind socket to address", se->data.client.fd);
+  char pbuffer[64];
```

Comment:
This variable currently used in `DEBUG()` call.
Then Collectd compiled w/o debug, this variable will be unused at practice - it will be filled by `inet_ntop()`, but not used later.

Mine proposal is to put it into `ERROR()` calls:

`ERROR("network plugin: failed to bind client socket (ipvX) to %s: %s", pbuffer, STRERRNO);`

This will use variable when non-debug version is compiled, and message become more informative.

Reply:
If you think that this is needed in non debug mode than I’ll change it.